### PR TITLE
Yaml perturbators

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

**Comment by jourdain** [comment]
@brianhhu 

I guess now we need someone to write the yaml to describe the parameters for any given perturber that we want to ingest inside NRTK explorer. 

That file can be loaded at runtime via an environment variable `NRTK_TRANSFORM_DEFINITION=/path/to/a/file.yaml`

File: src/nrtk_explorer/app/transforms.py [hunk 1]
```diff
@@ -153,6 +154,9 @@ def delete_meta_state(old_ids, new_ids):
             self._transforms["nrtk_blur"] = nrtk_trans.NrtkGaussianBlurTransform()
             self._transforms["nrtk_pybsm"] = nrtk_trans.NrtkPybsmTransform()
 
+        # Add transform from YAML definition
+        self._transforms.update(nrtk_yaml.generate_transforms())
```
Comment (PaulHax) [inline src/nrtk_explorer/app/transforms.py:158]: Erotemic tried to run once without `nrtk`, https://github.com/Kitware/nrtk-explorer/pull/91
Not sure why we want to support that, but there are checks for `nrtk_transforms_available` above.  I moved the line under the if statment in following PR.

**Review by PaulHax — APPROVED** [review]
💯

File: src/nrtk_explorer/app/transforms.py (hunk 1 above)
Comment (jourdain) [reply src/nrtk_explorer/app/transforms.py:158]: no need, the code already take care of that and skip any definition when the package is not found.

File: src/nrtk_explorer/library/yaml_transforms.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+# -----------------------------------------------------------------------------
+# Public
+# -----------------------------------------------------------------------------
+
+
+def generate_transforms():
+    transforms = {}
+    for k, v in TRANSFORM_DEFINITIONS.items():
+        with contextlib.suppress(ModuleNotFoundError):
```
Comment (jourdain) [inline src/nrtk_explorer/library/yaml_transforms.py:34]: See @PaulHax

**Pull request merged**